extensions/VisualEditor/wikia/extension.json

```diff
@@ -0,0 +1,38 @@
+{
+	"ResourceModules": {
+		"ext.visualEditor.wikia.desktopArticleTarget.init": {
+			"class": "VisualEditorDesktopArticleTargetInitModule",
+			"scripts": "ve/init/targets/ve.init.wikia.DesktopArticleTarget.init.js",
+			"styles": "ve/init/styles/ve.init.wikia.DesktopArticleTarget.init.css",
```

Comment:
Is it an issue that this css file doesn't exist? I imagine not since `addModuleStyles` is never called on this module